src/kakarot/account.cairo

```diff
@@ -126,14 +141,21 @@ namespace Account {
 
         let (bytecode_len, bytecode) = IAccount.bytecode(contract_address=starknet_address);
         let (nonce) = IAccount.get_nonce(contract_address=starknet_address);
+        IAccount.get_code_hash(contract_address=starknet_address);
```

Comment:
is there specific reason for not doing `let (code_hash) = IAccount.get_code_hash(contract_address=starknet_address);` ?

Reply:
note: I suggested this. In some case I don't manage to use `&` and this is a workaround